### ISSUE TITLE
ctx.actions.args() doesn't exist

### ISSUE THREAD

**Issue by aronasorman** [bazelbuild/bazel#3666]
Please provide the following information. The more we know about your system and use case, the more easily and likely we can help.

### Description of the problem / feature request / question:

```python
		ctx.actions.args()
type 'actions' has no method args().
```

I tried to create an `Args` instance, as documented [here](https://docs.bazel.build/versions/master/skylark/lib/Args.html).

Unfortunately, I get the erro above.

### Environment info
* Operating System:

MacOS. Bazel is installed via Homebrew.

* Bazel version (output of `bazel info release`):

```
$ bazel info release
release 0.5.4-homebrew
```

* If `bazel info release` returns "development version" or "(@non-git)", please tell us what source tree you compiled Bazel from; git commit hash is appreciated (`git rev-parse HEAD`):

###  Have you found anything relevant by searching the web?
(e.g. [StackOverflow answers](http://stackoverflow.com/questions/tagged/bazel),
[GitHub issues](https://github.com/bazelbuild/bazel/issues),
email threads on the [`bazel-discuss`](https://groups.google.com/forum/#!forum/bazel-discuss) Google group)

### Anything else, information or logs or outputs that would be helpful?
(If they are large, please upload as attachment or provide link).

**Comment by meteorcloudy** [comment]
@vladmos Any idea?

**Comment by vladmos** [comment]
It seems like it this functionality was submitted just a week ago (c2448d44a3ac673390a1d4916d80257ed6e8563e) and hasn't been released yet. It will be available in the upcoming Bazel 0.6 or if you build a custom Bazel binary with the aforementioned commit included.

**Comment by meteorcloudy** [comment]
Got it, so it would be better to update the document to clarify this functionality will be available from 0.6.
